src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Sequence.cs

```diff
@@ -24,6 +24,28 @@ public SequenceFunction()
         {
         }
 
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.SequenceArg1 };
+            yield return new[] { TexlStrings.SequenceArg1, TexlStrings.SequenceArg2 };
+            yield return new[] { TexlStrings.SequenceArg1, TexlStrings.SequenceArg2, TexlStrings.SequenceArg3 };
+        }
+    }
+
+    internal sealed class SequenceWFunction : BuiltinFunction
+    {
+        public override ArgPreprocessor GetArgPreprocessor(int index)
+        {
+            return base.GetGenericArgPreprocessor(index);
+        }
+
+        public override bool IsSelfContained => true;
+
+        public SequenceWFunction()
+            : base("Sequence", TexlStrings.AboutSequence, FunctionCategories.MathAndStat, DType.CreateTable(new TypedName(DType.Decimal, new DName("Value"))), 0, 1, 3, DType.Decimal, DType.Decimal, DType.Decimal)
```

Comment:
this doesn't seem right. Do we really need a separate overload of the sequence function?

Reply:
Let's chat.  If we don't and rely on floating point, and someone uses the output of Sequence in a calculation, then the whole calculation is tainted with floating point.